src/v/v8_engine/instance.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2021 Vectorized, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/vectorizedio/redpanda/blob/master/licenses/rcl.md
+ */
+
+#pragma once
+
+#include "seastarx.h"
+
+#include <seastar/core/future.hh>
+#include <seastar/core/sstring.hh>
+#include <seastar/core/temporary_buffer.hh>
+
+#include <v8.h>
+
+namespace v8_engine {
+
+class instance {
```

Comment:
Pls add class level doc-commetn. Mention what this instance represents (e.g. a V8 isolate + compiled script that can be executed)

Reply:
done